lib/kapnismology/smoke_test.rb

```diff
@@ -12,6 +12,10 @@ class SmokeTest
     RUNTIME_TAG = 'runtime'.freeze
     DEFAULT_TAGS = [DEPLOYMENT_TAG, RUNTIME_TAG].freeze
 
+    def initialize
```

Comment:
the warning is triggered because of https://github.com/JordiPolo/kapnismology/blob/master/lib/kapnismology/smoke_test.rb#L35 so I'll just add 
```
@all_result_messages ||= []
```
there as well then.